packages/demo/src/index.js

```diff
@@ -1,47 +1,112 @@
-import React from 'react';
```

Comment:
Can you remove these changes? We don't want to update the demo file. It should be enough to add the description in pdf-generator/Readme.md